.github/workflows/release.yml

```diff
@@ -88,14 +88,15 @@ jobs:
           wget https://apt.llvm.org/llvm.sh
           sed -i '/apt-get install/d' llvm.sh
           chmod +x llvm.sh
-          sudo ./llvm.sh 15
-          sudo apt-get install -y --no-install-recommends clang-15 lld-15 libunwind-15 libc++abi1-15 libc++1-15 libc++-15-dev
-          echo "build:linux --action_env=CC=/usr/lib/llvm-15/bin/clang --action_env=CXX=/usr/lib/llvm-15/bin/clang++" >> .bazelrc
-          echo "build:linux --host_action_env=CC=/usr/lib/llvm-15/bin/clang --host_action_env=CXX=/usr/lib/llvm-15/bin/clang++" >> .bazelrc
+          sudo ./llvm.sh 18
+          sudo apt-get install -y --no-install-recommends clang-18 lld-18 libunwind-18 libc++abi1-18 libc++1-18 libc++-18-dev
+          echo "build:linux --action_env=CC=/usr/lib/llvm-18/bin/clang --action_env=CXX=/usr/lib/llvm-18/bin/clang++" >> .bazelrc
+          echo "build:linux --host_action_env=CC=/usr/lib/llvm-18/bin/clang --host_action_env=CXX=/usr/lib/llvm-18/bin/clang++" >> .bazelrc
       - name: Setup macOS
         if: runner.os == 'macOS'
         run: |
-          sudo xcode-select -s "/Applications/Xcode_15.1.app"
+          brew install llvm@18
+          export PATH="$(brew --prefix llvm@18)/bin/bin:$PATH"
```

Comment:
We have always built workerd with the Xcode-provided clang/LLVM toolchain. Apple LLVM generally has some differences from mainline LLVM, and with the way toolchain resolution works in https://github.com/bazelbuild/apple_support I think that bazel will always build using Xcode's LLVM even if this is added to PATH. Building with a different LLVM may come with unexpected consequences just based on the differences Apple Clang has, currently Apple Clang 15 is based on LLVM 16 which feels recent enough.

Reply:
I have been building workerd locally with brew llvm@18 with no issues but as with the previous comment, I'll just revert this entire change.